composer.json

```diff
@@ -38,7 +38,8 @@
     },
     "require-dev" : {
         "friendsofphp/php-cs-fixer": "~2.16.1",
-        "phpunit/phpunit" : "^7 || ^8"
+        "phpunit/phpunit" : "^7 || ^8",
+        "phpstan/phpstan": "^0.12"
```

Comment:
@staabm I know that you wanted me to keep `phpstan` out of `composer.json`
But maybe that was just while we had PHP 7.0 support, and `phpstan` had to have a different version depending on the PHP version ...

Reply:
as long as phpstan does not bump phpstan min version this should be fine